.jules/bin/build_index.py

```
@@ -1,27 +1,30 @@
     output_file = repo_root / ".jules" / "index" / "generated" / "RUNS_ROLLUP.md"
 
     runs = []
+
+    # Process standard .jules/runs packets
     if runs_dir.exists():
         for run_path in sorted(runs_dir.iterdir(), key=lambda path: path.name):
             if not run_path.is_dir():
                 continue
 
             envelope = read_json(run_path / "envelope.json")
+            if not isinstance(envelope, dict): envelope = {}
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)